docs/commands/push-status.md

```diff
@@ -0,0 +1,64 @@
+
+The `push-status` command can be used whenever the application or process executing a `push` command (without `--wait-for-deployment` option) returns the `pushId`.
+This identifier can be used by subsequent systems to perform custom logic when the deployment is completed.
+
+## Prerequisites
+
+Have the following before you use the `push-status` command:
+
+- A user account in a [Reunite project](https://auth.cloud.redocly.com/).
+- An active organization [API key](https://redocly.com/docs/realm/setup/how-to/api-keys)
+- [Redocly CLI](../installation.md) v1.10.x or later.
+
+## Authentication
+
+Use the `REDOCLY_AUTHORIZATION` environment variable to set the API key.
+See [Manage API keys](https://redocly.com/docs/realm/setup/how-to/api-keys) page for details on how to get your API key in Reunite.
+
+## Usage
+
+```bash
+REDOCLY_AUTHORIZATION=<api-key> redocly push-status <pushId> --organization <orgSlug> --project <projectSlug> [--wait] [--max-execution-time <timeInSeconds>]
+```
+
+## Options
+
+| Option               | Type    | Required | Default value                                                  | Description                                                                                                   |
+| -------------------- | ------- | -------- |----------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|
+| pushId               | string  | true     | -                                                              | Identifier of the push whose state should be displayed. Returned as result of the [`push`](./push.md) command. |
+| --organization, -o   | string  | true     | -                                                              | [Organization slug](#where-to-find-the-redocly-organization-or-project-slugs).                                 |
+| --project, -p        | string  | true     | -                                                              | [Project slug](#where-to-find-the-redocly-organization-or-project-slugs).                                      |
+| --domain, -d         | string  | false    | [https://app.cloud.redocly.com](https://app.cloud.redocly.com) | The domain that the `push` command pushed to.                                                                  |
+| --wait               | boolean | false    | false                                                          | Waits until the build is completed if it is in progress.                                                       |
+| --max-execution-time | number  | false    | 600                                                            | Maximum wait time for build completion in seconds (used in conjunction with the `--wait` option).              |
+
+### Where to find the Redocly organization or project slugs
+
+{% partial file="../../../_partials/get-organization-and-project-slugs.md" /%}
```

Comment:
There are some issues with the partial that I am going to address in a corresponding PR. I will also look into making the section linkable so we don't need to include a partial.

Reply:
The corresponding PR is https://github.com/Redocly/redocly/pull/10002

Reply:
Thanks!